penrose-web/src/types/types.d.ts

```diff
@@ -1447,8 +1629,50 @@ interface ASTNode {
 
 interface Identifier extends ASTNode {
   tag: "Identifier";
-  type: string;
-  value: string;
+  type: string; // meta-info: either `value` or `type-identifier` according to the parser
+  value: string; // the actual value
+}
+
+//#endregion
+
+//#region
+
+type StyErrors = string[];
+// TODO: Convert this to StyleError[]
+
+interface Left<A> {
```

Comment:
Use true-myth's either (i.e. `Result`) instead? Doesn't really matter if `Either` is only internal to the Style compiler. If so, it'll make more sense for them to be defined in the module itself?